x-pack/plugins/serverless/public/plugin.tsx

```diff
@@ -65,6 +65,8 @@ export class ServerlessPlugin
       setSideNavComponent: (sideNavigationComponent) =>
         // Casting the "chrome.projects" service to an "internal" type: this is intentional to obscure the property from Typescript.
         (core.chrome as InternalChromeStart).project.setSideNavComponent(sideNavigationComponent),
+      setBreadcrumbs: (breadcrumbs, params) =>
+        (core.chrome as InternalChromeStart).project.setBreadcrumbs(breadcrumbs, params),
```

Comment:
Nit: maybe the comment on line 66 should be moved up since it refers to this function as well.